mnefun/_mnefun.py

```diff
@@ -1730,12 +1731,13 @@ def do_preprocessing_combined(p, subjects, run_indices):
                   '        %s' % bad_file)
             if not op.isdir(bad_dir):
                 os.mkdir(bad_dir)
-            # do autobad
+            # do autobad without applying eeg average ref projection
```

Comment:
Ok. Is there a way in mne python to delay application of the projector (specifically the eeg avg ref)? otherwise it sounds like we might need a helper file to do this.